pkg/install/config.go

```diff
@@ -0,0 +1,46 @@
+package install
+
+import (
+	"time"
+
+	arkv1 "github.com/heptio/ark/pkg/apis/ark/v1"
+
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+func Config(
+	namespace string,
+	pvCloudProviderName string,
+	pvCloudProviderConfig map[string]string,
+	backupCloudProviderName string,
+	backupCloudProviderConfig map[string]string,
+	bucket string,
+) *arkv1.Config {
+	return &arkv1.Config{
+		ObjectMeta: metav1.ObjectMeta{
+			Namespace: namespace,
+			Name:      "default",
+		},
+		PersistentVolumeProvider: &arkv1.CloudProviderConfig{
+			Name:   pvCloudProviderName,
+			Config: pvCloudProviderConfig,
+		},
+		BackupStorageProvider: arkv1.ObjectStorageProviderConfig{
+			CloudProviderConfig: arkv1.CloudProviderConfig{
+				Name:   backupCloudProviderName,
+				Config: backupCloudProviderConfig,
+			},
+			Bucket: bucket,
+		},
+		BackupSyncPeriod: metav1.Duration{
+			Duration: 30 * time.Minute,
```

Comment:
It would be nice to have the sync periods, and restoreonlymode, be configurable. Use the variadic config functions pattern here?

Reply:
I would be happy to contribute this, but such an API change would be completely additive. Thus, would it be possible to deliver this in another follow-up PR?

Reply:
Follow-up is fine with me.